src/sidebar/services/frame-sync.js

```diff
@@ -8,29 +8,33 @@ import { isReply, isPublic } from '../helpers/annotation-metadata';
 import { watch } from '../util/watch';
 
 /**
+ * @typedef {import('../../types/annotator').AnnotationData} AnnotationData
+ * @typedef {import('../../types/api').Annotation} Annotation
  * @typedef {import('../../types/bridge-events').SidebarToHostEvent} SidebarToHostEvent
  * @typedef {import('../../types/bridge-events').HostToSidebarEvent} HostToSidebarEvent
  * @typedef {import('../../types/bridge-events').SidebarToGuestEvent} SidebarToGuestEvent
  * @typedef {import('../../types/bridge-events').GuestToSidebarEvent} GuestToSidebarEvent
  * @typedef {import('../../shared/port-rpc').PortRPC} PortRPC
+ * @typedef {import('../store/modules/frames').Frame} Frame
  */
 
 /**
  * Return a minimal representation of an annotation that can be sent from the
- * sidebar app to a connected frame.
+ * sidebar app to the host frame.
  *
  * Because this representation will be exposed to untrusted third-party
  * JavaScript, it includes only the information needed to uniquely identify it
  * within the current session and anchor it in the document.
+ *
+ * @param {Annotation} annotation
+ * @returns {AnnotationData}
  */
-export function formatAnnot(ann) {
+export function formatAnnot({ $tag, document, target, uri }) {
   return {
-    tag: ann.$tag,
-    msg: {
-      document: ann.document,
-      target: ann.target,
-      uri: ann.uri,
-    },
+    $tag,
+    document: { title: document.title, link: [{ href: target[0].source }] },
```

Comment:
I needed to format it a little the returned object, to make it fully compatible with `AnnotationData`.